### ISSUE TITLE
Importing tweets fail

### ISSUE THREAD

**Issue by coggy9** [martani/Twitter-Archive-Eraser#7]
I'm trying to import my archive/tweets.js file, but the app can't parse the file. 

It fails to load dates or any other info except the IDs. It might have to do with the size of the file, or Twitter changing their format.

![2018-07-14_01-19-31](https://user-images.githubusercontent.com/352207/42721665-2c60ca24-8704-11e8-9379-785beb8ccd21.png)

![twitter archive eraser_2018-07-14_01-16-14](https://user-images.githubusercontent.com/352207/42721668-342b6e9e-8704-11e8-8925-ebb90dea28b1.png)

**Comment by martani** [comment]
Just import the whole zip file. It should work.

**Comment by coggy9** [comment]
I tried to import the unmodified ZIP, but it gave me an error saying "No Twitter archive or *.js files were loaded!"

**Comment by martani** [comment]
Can you try to re-download your archive? If that doesn't work. Can you please share your archive via email for further investigation?

**Comment by coggy9** [comment]
It's been a few weeks since I made this one, so I'll have to request a new archive. If the new one also doesn't work, I'll send it over.

**Comment by coggy9** [comment]
Redownloading gives the same error. Where should I send the archive to?

**Comment by martani** [comment]
http://www.martani.net/p/contact-me.html

**Comment by swxxii** [comment]
I'm getting this error too.

**Comment by melissamcewen** [comment]
me as well

**Comment by melissamcewen** [comment]
it appears the archives are not quite uniform which is causing issues on similar projects https://twitter.com/ossia/status/1021431097060716544

**Comment by martani** [comment]
### TLDR
The correct format is the one from: https://twitter.com/settings/account (see https://help.twitter.com/en/managing-your-account/how-to-download-your-twitter-archive)

_Do not use the archive from "Your Twitter data" (https://help.twitter.com/en/managing-your-account/accessing-your-twitter-data) or wait for the next version._

**Comment by swxxii** [comment]
This fixed the issue. Thanks GDPR!
